apps/web/src/components/Profile/components/WorkPreferences/FormFields.tsx

```diff
@@ -51,6 +60,50 @@ const FormFields = ({
   const data = getFragment(WorkPreferencesFormOptions_Fragment, optionsQuery);
   useDirtyFields("work");
 
+  // taking the enum collection from the API request
+  // turn them into options, append a content-below node
+  const formOptionsWithContentBelow: CheckboxOption[] =
+    sortFlexibleWorkLocations(data?.flexibleWorkLocation).map((loc) => {
+      if (loc.value === (FlexibleWorkLocation.Remote as string)) {
+        return {
```

Comment:
Not beautiful, but had to handle the type business with API options -> localized options then compare them to enums
`FlexibleWorkLocation.Remote` is basically the same as `"REMOTE"` after all

Reply:
I think this is fine.